classes/VersionUtils.php

```diff
@@ -96,6 +96,17 @@ public static function isActualPHPVersionCompatible()
         return PHP_VERSION_ID >= self::MODULE_COMPATIBLE_PHP_VERSION;
     }
 
+    /**
+     * @param string $version
+     * 
+     * @return string|null
+     */
+    public static function getPrestashopMajorVersion($version)
```

Comment:
@M0rgan01 The extracted method added is not totally aligned with the one below in terms of exceptions thrown.

There is likely some refacto to do as well, as we are using two different methods to extract separate pieces of the same variable.
We could have a single method returning `array{'minor' => string, 'major' => string}`

Reply:
Yes I see that, we could totally have one and the same method; We use the other to calculate compatibility ranges. I could merge them